### PR TITLE
Fix existing unit test, switch to pytest, add more

### PR DESCRIPTION
**Pull request by endolith**
Are you committed to the `unittest` framework?  I'm used to `pytest` and not sure what the equivalent of `@pytest.mark.parametrize` is.  

I also found [hypothesis](https://hypothesis.readthedocs.io/en/latest/) very useful in finding broken edge cases in my implementation, if you're ok with importing that.  https://github.com/endolith/elsim/blob/master/tests/test_star.py#L550

### CODE REVIEW

**Comment by masiarek** [comment]
I prefer pytest as well..

**Comment by mikefranze** [comment]
I'm not committed to any framework. You can change to pytest if you prefer it.

**Comment by mikefranze** [comment]
My last PR also fixed the existing unit test and also changed the output format.

File: STAR_Test.py [hunk 1]
```diff
@@ -19,13 +19,26 @@ def test_original_example(self):
...
+        # https://en.wikipedia.org/wiki/STAR_voting#Example
+        # https://electowiki.org/wiki/STAR_voting#Example
+        columns = ['Memphis', 'Nashville', 'Chattanooga', 'Knoxville']
+        ballots = pd.DataFrame(columns=columns,
+                               data=[*42*[[5,      2,        1,          0]],
+                                     *26*[[0,      5,        2,          1]],
+                                     *15*[[0,      3,        5,          3]],
+                                     *17*[[0,      2,        4,          5]]])
+
+        self.assertEqual(STAR(ballots), 'Nashville')
```
Comment (mikefranze) [inline STAR_Test.py]: Does this work? I think you need to pull out elected candidates from the results: self.assertEqual(results['elected'] , ['Nashville'])

Comment (endolith) [reply STAR_Test.py]: I had just rebased on top of the new output format without changing it. It should be fixed now

**Review by mikefranze — APPROVED** [review]

**Review by ArendPeter — APPROVED** [review]
Looks great! I'm also a pytest fan
